pkg/skaffold/debug/transform_go.go

```diff
@@ -61,10 +61,27 @@ func isLaunchingDlv(args []string) bool {
 func (t dlvTransformer) IsApplicable(config imageConfiguration) bool {
 	for _, name := range []string{"GODEBUG", "GOGC", "GOMAXPROCS", "GOTRACEBACK"} {
 		if _, found := config.env[name]; found {
+			logrus.Infof("Artifact %q has Go runtime: has env %q", config.artifact, name)
 			return true
 		}
 	}
-	if len(config.entrypoint) > 0 {
+
+	// FIXME: as there is currently no way to identify a buildpacks-produced image as holding a Go binary,
+	// nor to cause certain environment variables to be defined in the resulting image, look at the image's
+	// CNB metadata to see if any well-known Go-related buildpacks had been involved.
+	knownGoBuildpackIds := []string{
+		"google.go.runtime", "google.go.build", "google.go.gopath", "google.go.gomod", // GCP Buildpacks
```

Comment:
`google.go.build` is always on, all the other on that line are optional ans can't run without `google.go.build`